Modified docker documentation (#233)
Percona-Lab/pg_tde: documentation/docs/install.md

    docker run --name pg-tde -e POSTGRES_PASSWORD=mysecretpassword -d perconalab/pg_tde
    ```    

>     It builds and adds `pg_tde` extension to PostgreSQL 16. The `postgresql.conf` contains the required modifications, and the extension is added to `template1` - with this, all new databases automatically have the `pg_tde` extension loaded. 

Reviewer: Don't we mention where the postgresql.conf is stored inside the container anymore?

Author: That's a feature of the "upstream" docker image - do we want to mention it?

Reviewer: We just mentioned it before. If it's not important / well known for Docker users/ brings no value to the user, we can omit it

>     It builds and adds `pg_tde` extension to PostgreSQL 16. The `postgresql.conf` contains the required modifications, and the extension is added to `template1` - with this, all new databases automatically have the `pg_tde` extension loaded. 

Reviewer: ```suggestion
    It builds and adds `pg_tde` extension to PostgreSQL 16. The `postgresql.conf` contains the required modifications. The `pg_tde` extension is added to `template1` so that all new databases automatically have the `pg_tde` extension loaded. 
```

>     Keys are not created automatically, each database requires a key provider and a principal key before encrypted tables can be used. For that, see the instructions in the [Setup](setup.md) section, starting with the 4th point, as the first 3 steps are already completed in the docker image.

Reviewer: ```suggestion
    Keys are not created automatically. You must configure a key provider and a principal key for each database  where you wish to use encrypted tables. See the instructions in the [Setup](setup.md) section, starting with the 4th point, as the first 3 steps are already completed in the Docker image.
```
